### ISSUE TITLE
Discovery result sort options not reflecting what is configured

### ISSUE THREAD

**Issue by corrad82-4s** [DSpace/dspace-angular#1071]
**Describe the bug**
The "sort" select box displayed in the discovery results page contains by default "Relevance", "Title Ascending" and "Title Descending" as possible "sort by" options, instead of options that are defined in discovery.xml file. 

**To Reproduce**
1. login 
2. go to "My DSpace" page
3. check "sort by" select 
4. Displayed options are different than what set into discovery for "workspace" configuration (currently "sortTitle" and "sortDateIssued")

**Expected behavior**
A select with options that allow to set sorting by both title and dateIssued

We 4Science are available to look into this issue

**Comment by tdonohue** [comment]
@corrad82-4s : If you (or someone at 4Science) could provide an estimate for this, then I can more easily make a decision on whether we can fit it into Beta 5.  Thanks!

**Comment by corrad82-4s** [comment]
Hi @tdonohue we estimate that this issue can be resolved in 4 to 6 hours of work. Thanks

**Comment by tdonohue** [comment]
@corrad82-4s : that sounds reasonable. I'll tentatively add this to Beta 5.

As a sidenote, any work on this ticket should be based on the work in #1048, since that affects pagination of search results in general across the UI.  So, please make sure to review that work first before starting a new PR... I don't believe that work fixes this bug, but it might affect your fix.

**Comment by tdonohue** [comment]
Once Angular PR is ready, assign to me & @artlowel to review.
